### ISSUE TITLE
puppetlabs-ruby required version bump

### ISSUE THREAD

**Issue by den-is** [DataDog/puppet-datadog-agent#353]
```
./datadog_agent/metadata.json:12:    {"name":"puppetlabs/ruby","version_requirement":">=0.2.0 <1.0.0"},
```
current version of ruby module is 1.0.0
please bump version requirement.

**Comment by truthbk** [comment]
@den-is I'll take a look at this. We'll have to ensure this doesn't break anything for older puppets.

**Comment by bittner** [comment]
> We'll have to ensure this doesn't break anything for older puppets.

@truthbk, we can [test against older versions](https://travis-ci.org/DataDog/puppet-datadog-agent) of Puppet, can't we? So, bumping a version of a package is mostly a trial-and-error game.

**Comment by truthbk** [comment]
This was addressed with the release of `2.0`. Thank you @den-is.
